src/containers/ProductListing.js

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from 'react';
+import axios from 'axios';
+import { useDispatch, useSelector } from 'react-redux';
+import { setProducts } from '../redux/actions/productsActions';
+import ProductComponent from './ProductComponent';
+
+const URL = 'https://fakestoreapi.com/products';
+
+const ProductPage = () => {
+  const products = useSelector((state) => state.allProducts.products);
+  const dispatch = useDispatch();
+  const getProducts = async () => {
+    const response = await axios
+      .get(URL)
+      .catch((err) => {
+        console.log('Err: ', err);
+      });
+    dispatch(setProducts(response.data));
+  };
+
+  useEffect(() => {
+    getProducts();
+  }, []);
+
+  console.log('Products :', products);
+  return (
+    <div className="ui grid container">
+      <ProductComponent />
+    </div>
```

Comment:
- [ ] I think it would be great if you follow the design here to make it the same so kindly make the navbar since I can't see it in your  home page💯🚀

- Your design
![image](https://user-images.githubusercontent.com/58553711/203681529-ac2a974b-1c8b-47e2-ae94-216f3451685e.png)

- Original design
![image](https://user-images.githubusercontent.com/58553711/203681651-b72e62dc-126e-4f45-be1f-14eb4fc46166.png)

Reply:
- As requested by the previous reviewer,  please kindly stick to the background color as shown above to improve your design and to match the design as this is a core requirement to have your project approved.

Reply:
- [ ] This issue is not yet resolved :x:

Reply:
- [ ] This issues is not yet resolved ❌